api/tests/connectors/harvestr_test.py

```diff
@@ -2,11 +2,10 @@
 from requests import HTTPError
 
 from pcapi.connectors import harvestr
-from pcapi.core.testing import override_settings
 
 
-@override_settings(ENV="pro")
-@override_settings(PRO_URL="https://passculture.pro")
+@pytest.mark.settings(ENV="pro")
+@pytest.mark.settings(PRO_URL="https://passculture.pro")
```

Comment:
```suggestion
@pytest.mark.settings(ENV="pro",PRO_URL="https://passculture.pro")
```
Je crois que sinon ça marche pas 😓

Reply:
Ah ^^ 
j'update 👍